Boolean.CSharp.Test/CoreTests.cs

```diff
@@ -1,24 +1,125 @@
-﻿using Boolean.CSharp.Main;
+﻿using System.Security.Principal;
+using Boolean.CSharp.Main;
 using NUnit.Framework;
 
 namespace Boolean.CSharp.Test
 {
-    [TestFixture]
     public class CoreTests
     {
-        private Core _core;
 
-        public CoreTests()
+
+
+        [Test]
+        public void addCustomer()
+        {
+            Bank bank = new Bank();
+            Customer bob = new Customer("Bob");
+            bool customerTest = bank.addCustomer(bob);
+
+            Assert.IsTrue(customerTest);
+
+        }
+
+        [Test]
+        public void addAccountToCustomer()
+        {
+            Bank bank = new Bank();
+            Customer bob = new Customer("Bob");
+            Account current = new Account("Current");
+            bank.addCustomer(bob);
+            bool addAccountTest = bank.addAccountToCustomer("Bob", current);
+
+            Assert.IsTrue(addAccountTest);
+
+        }
+
+        [Test]
+        public void addMoneyToCurrent()
+        {
+            Bank bank = new Bank();
+            Customer bob = new Customer("Bob");
+            Account current = new Account("Current");
+            bank.addCustomer(bob);
+            bank.addAccountToCustomer("Bob", current);
+            bool addMoneyTest = bank.depositMoney("Bob", "Current", 2000f, "14/01/2012");
+
+
+            Assert.IsTrue(addMoneyTest);
```

Comment:
not enough assertions, you should check that the transaction was created and that it contains the right values and that the account balance was updated...